### PR TITLE
Fix debug code in progressbar.dm

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
exceptions need to be thrown

### CODE REVIEW

File: code/datums/progressbar.dm [hunk 1]
```diff
@@ -21,7 +21,7 @@
 /datum/progressbar/New(mob/User, goal_number, atom/target)
 	. = ..()
 	if (!istype(target))
-		EXCEPTION("Invalid target given")
+		throw EXCEPTION("Invalid target given")
```
Comment (LemonInTheDark) [inline code/datums/progressbar.dm:24]: Nothing is setup to catch this, it should be a stacktrace + qdel rather then an exception throw
```suggestion
		stack_trace("Invalid target [target] passed in")
		qdel(src)
		return
```

Comment (Kapu1178) [reply code/datums/progressbar.dm:24]: world/Error automatically catches it, and it's automatically placed into the runtime viewer. This pr was infact an excuse to share that knowledge.

Comment (LemonInTheDark) [reply code/datums/progressbar.dm]: damn you

Comment (Kapu1178) [reply code/datums/progressbar.dm]: L

**Review by spookydonut — APPROVED** [review]

**Pull request merged**